simulation/sub8_montecarlo/nodes/example_verification.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+'''Monte-Carlo controller verification
+'''
+import rospy
+from sub8_montecarlo_tools import VerifyController
+
+
+if __name__ == '__main__':
+    rospy.init_node('verify_pd')
```

Comment:
Does this really need to be in a separate file? It looks like this could be moved into a comment in controller_verify.py. This code is already being ran in your PD controller test as well; an option is to write a separate unit test for this basic use case of the VC class though.